Packs/Rapid7_Nexpose/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Rapid7 InsightVM",
     "description": "Vulnerability management solution to help reduce threat exposure.",
     "support": "xsoar",
-    "currentVersion": "1.2.33",
+    "currentVersion": "1.2.34",
```

```suggestion
    "currentVersion": "1.3.0",
```
Same for release notes file